src/patternfly/components/InlineEdit/examples/InlineEdit.md

```diff
@@ -0,0 +1,563 @@
+        {{/inline-edit-input}}
+        {{#> inline-edit-input}}
+          {{#> radio}}
+            {{#> radio-input radio-input--attribute=(concat 'id="' inline-edit--id '-radio2" name="' inline-edit--id '-input"')}}{{/radio-input}}
+            {{#> radio-label radio-label--attribute=(concat 'for="' inline-edit--id '-radio2"')}}False{{/radio-label}}
+          {{/radio}}
+        {{/inline-edit-input}}
+      {{/inline-edit-group}}
+    </dd>
+  {{/inline-edit}}
+
+  {{#> inline-edit inline-edit--id="inline-edit-dl-example-4" inline-edit--modifier="pf-m-inline-editable"}}
+    <dt>
+      {{#> inline-edit-group}}
+        {{#> inline-edit-label}}
+          Value
+        {{/inline-edit-label}}
+        {{#> inline-edit-toggle}}
+          {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'id="' inline-edit--id '-edit-button" aria-label="Edit" aria-labelledby="' inline-edit--id '-label ' inline-edit--id '-edit-button"')}}
+            <i class="fas fa-pencil-alt" aria-hidden="true" />
+          {{/button}}
+        {{/inline-edit-toggle}}
+      {{/inline-edit-group}}
+    </dt>
+    <dd>
+      {{#> inline-edit-value}}
+        True
+      {{/inline-edit-value}}
+      {{#> inline-edit-group inline-edit-group--attribute='role="radiogroup" aria-label="Radio group example"' inline-edit-group--modifier="pf-m-column"}}
+        {{#> inline-edit-input}}
+          {{#> radio}}
+            {{#> radio-input radio-input--attribute=(concat 'id="' inline-edit--id '-radio1" name="' inline-edit--id '-input"')}}{{/radio-input}}
+            {{#> radio-label radio-label--attribute=(concat 'for="' inline-edit--id '-radio1"')}}True{{/radio-label}}
+          {{/radio}}
+        {{/inline-edit-input}}
+        {{#> inline-edit-input}}
+          {{#> radio}}
+            {{#> radio-input radio-input--attribute=(concat 'id="' inline-edit--id '-radio2" name="' inline-edit--id '-input"')}}{{/radio-input}}
+            {{#> radio-label radio-label--attribute=(concat 'for="' inline-edit--id '-radio2"')}}False{{/radio-label}}
+          {{/radio}}
+        {{/inline-edit-input}}
+      {{/inline-edit-group}}
+    </dd>
+  {{/inline-edit}}
+  <!-- tags, pods, value -->
+{{/list}}
+```
+
+```hbs title=Inline-edit-table-row
+{{#> inline-edit inline-edit--type="form" inline-edit--id="bulk-edit-table-example"}}
+  {{#> table table--grid="true" table--modifier="pf-m-grid-lg" table--attribute='aria-label="Inline edit table row example"'}}
+    {{#> table-caption}}
+      This is the table caption
+    {{/table-caption}}
+    {{#> table-thead}}
+      {{#> table-tr}}
+        {{#> table-th}}
+          Text input
+        {{/table-th}}
+        {{#> table-th}}
+          Disabled text input
+        {{/table-th}}
+        {{#> table-th}}
+          Checkboxes
+        {{/table-th}}
+        {{#> table-th}}
+          Radios
+        {{/table-th}}
+        {{#> table-th}}
+          Number
+        {{/table-th}}
+        {{#> table-td}}{{/table-td}}
```

Comment:
this should be updated to reflect the changes we made to the table: table-td cell is empty